web/src/components/warning-banner/index.tsx

```diff
@@ -35,7 +35,7 @@ export const WarningBanner: FC<WarningBannerProps> = memo(
     return (
       <AppBar position="static" className={classes.container}>
         <div className={classes.content}>
-          Version {process.env.PIPECD_VERSION} is available! Find all{" "}
```

Comment:
How about this?

```
        <div className={classes.content}>
          Piped version {process.env.PIPECD_VERSION}
          <a
            href={releaseNoteURL}
            target="_blank"
            rel="noreferrer"
            className={classes.highlight}
          >
            {process.env.PIPECD_VERSION}
          </a>
          is available!
        </div>
```

Reply:
![Screen Shot 2022-05-27 at 13 05 11](https://user-images.githubusercontent.com/32532742/170640323-4bfff6fb-b13f-4392-96d1-197a87dacbbd.png)
It would look like this 🙌

Reply:
Nice!